config/pmix.m4

```diff
@@ -301,7 +301,8 @@ AC_DEFUN([PMIX_SETUP_CORE],[
                       time.h unistd.h dirent.h \
                       crt_externs.h signal.h \
                       ioLib.h sockLib.h hostLib.h limits.h \
-                      sys/statfs.h sys/statvfs.h])
+                      sys/statfs.h sys/statvfs.h \
+                      netdb.h ucred.h])
```

Comment:
Ah, okay - I didn't see you add something and didn't realize it was missing. Thanks